front/src/modules/conflict/components/ConflictCardV2.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+import { useTranslation } from 'react-i18next';
+import { BsLightningFill } from 'react-icons/bs';
+
+import type { ConflictV2 } from 'common/api/osrdEditoastApi';
+import type { TrainScheduleWithDetails } from 'modules/trainschedule/components/TimetableV2/types';
+import { stringToTime } from 'utils/date';
+
+export type ConflictTableProps = {
+  conflicts: ConflictV2[];
+  expanded: boolean;
+  trainSchedulesDetails: TrainScheduleWithDetails[];
+  toggleConflictsList: () => void;
+  onClick: (conflict: ConflictWithTrainNames) => void;
```

Comment:
Seems more descriptive to me
```suggestion
  onConflictClick: (conflict: ConflictWithTrainNames) => void;
```

Reply:
done